README.md

```diff
@@ -1,15 +1,16 @@
 # sourmash
 
+Quickly search, compare, and analyze genomic and metagenomic data sets.
+
 [![Documentation](https://readthedocs.org/projects/sourmash/badge/?version=latest)](http://sourmash.readthedocs.io/en/latest/)
 [![Build Status](https://travis-ci.com/dib-lab/sourmash.svg?branch=master)](https://travis-ci.com/dib-lab/sourmash)
+[![Bioconda install](https://img.shields.io/conda/dn/bioconda/sourmash.svg?style=flag&label=Bioconda)](https://anaconda.org/bioconda/sourmash)
 <a href="https://pypi.org/project/sourmash/"><img alt="PyPI" src="https://badge.fury.io/py/sourmash.svg"></a>
 [![codecov](https://codecov.io/gh/dib-lab/sourmash/branch/master/graph/badge.svg)](https://codecov.io/gh/dib-lab/sourmash)
 [![DOI](http://joss.theoj.org/papers/10.21105/joss.00027/status.svg)](http://joss.theoj.org/papers/10.21105/joss.00027)
 <a href="https://github.com/dib-lab/sourmash/blob/master/LICENSE"><img alt="License: 3-Clause BSD" src="https://img.shields.io/badge/License-BSD%203--Clause-blue.svg"></a>
 
----
-
-Quickly search, compare, and analyze genomic and metagenomic data sets.
+<p align="center"><img src="https://raw.githubusercontent.com/dib-lab/sourmash/logo/doc/_static/logo.png" height="256" /></p>
```

Comment:
Need to fix this link before merge (point to `master`, not the `logo` branch), but this way it is possible to see how it looks.